tests/e2e/pageobjects/ide/OpenDialogWidget.ts

```diff
@@ -20,13 +20,13 @@ import { TimeoutConstants } from '../../TimeoutConstants';
 export enum Locations {
     Theia = 'theia',
     Home = 'home',
-    Root = '/'
+    Root = '/',
 }
 
 export enum Buttons {
     Cancel = 'Cancel',
     AddContext = 'Add context folder for component in workspace.',
-    Open = 'Open'
+    Open = 'Open',
```

Comment:
@makambalaji this doesn't look right. There are no more variables after this one, the `,` shouldn't be there.